src/utils/utils.ts

```diff
@@ -78,9 +80,10 @@ export const getAriaAttrs = ({
         ? `${labelText} - ${footnote}`
         : (labelText as string);
   } else if (footnote) {
-    ariaAttributes["aria-describedby"] = `${id}-helperText`;
+    ariaAttributes["aria-describedby"] = `${
+      secondaryHelperTextId ? secondaryHelperTextId + " " : ""
```

Comment:
This fixes the issue but _if_ we need to later add more ids to this string list, then this will become unmanageable with more props and logic checks because the name implies it's a single additional id. Perhaps this can be renamed to `additionalHelperTextIds` and specify that it can be a single id or a space-separated string of ids (similar to how aria-describedby is documented).

Reply:
That's a good suggestion.  

@oliviawongnyc Is this something you can do?  This goes back to that idea of making the component as flexible as possible.

Reply:
This is done!